pss/pss.go

```diff
@@ -52,11 +61,27 @@ func (p *Pss) Send(ctx context.Context, targets [][]byte, topic trojan.Topic, pa
 		return nil, err
 	}
 
+	tc.WithTagID(p.tag.Uid)
+
 	// SAVE trojanChunk to localstore, if it exists do nothing as it's already peristed
-	// TODO: for second phase, use tags --> listen for response of recipient, recipient offline
 	if _, err = p.localStore.Put(ctx, chunk.ModePutUpload, tc); err != nil {
 		return nil, err
 	}
+	p.tag.Inc(chunk.StateStored)
 
 	return tc, nil
 }
+
+// GetState return the state of a pss message sent
```

Comment:
could you please explain what this message would be? (in order to understand better the purpose of this func)

Reply:
removed, addressed in [here](https://github.com/ethersphere/swarm/pull/2159#discussion_r409565485)